org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/StressTestSuspendVMConditionalBreakpoints.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.jdt.debug.tests;
```

Comment:
Will likely be kicking out this file from the final version of the commit, so no copyright header.

Also 500 repeats is exaggerated, I got a deadlock already in the first few test runs.